TBOOMDetector/TBOOMDetector.m

```diff
@@ -25,13 +25,12 @@ @implementation TBOOMDetector
 static NSString *AppVersionKey = @"AppVersion";
 static NSString *OSVersionKey = @"OSVersion";
 
-- (instancetype)initWithCrashlyticsApiKey:(NSString*)apiKey
-                                directory:(NSString*)directory
-                                 callback:(void (^)(TBTerminationType terminationType))callback {
+- (instancetype)initWithDirectory:(NSString*)directory
+                       crashCheck:(BOOL (^)(void))crashCheck
+                         callback:(void (^)(TBTerminationType terminationType))callback {
   self = [super init];
   if(self) {
     stateDirectory = directory;
-    [Crashlytics startWithAPIKey:apiKey delegate:self];
```

Comment:
I'm assuming removing this means whoever calls TBOOMDetector and passes in a crashCheck block is responsible for initializing their crash-checker ahead of time... which makes sense.